alertmanager/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Stage1: build from source
 FROM ghcr.io/cybozu/golang:1.21-jammy AS build
```

Comment:
Have you tried golang 1.22 ?
(see section 3 of https://neco.cybozu.wiki/procedure_regular_update/#_11)

Reply:
thank you for your comment, I will try this.